aws/iam/iam-policy-for-taggable-resources/locals.tf

```
@@ -11,7 +11,10 @@ locals {
     "elasticloadbalancing",
     "elasticache",
     "rds",
-    "s3"
+    "s3",
+    "waf",
+    "wafv2",
+    "waf-regional"
```

```suggestion
    "waf-regional",
```

nit: lowers risk that we update the list with a syntax error. Also simpler diffs when extending the list.